PontoFacil/PontoFacil/Views/SettingsPage.xaml.cs

```diff
@@ -25,6 +15,9 @@ public sealed partial class SettingsPage : Page
         public SettingsPage()
         {
             this.InitializeComponent();
+            ApplicationView.GetForCurrentView().SetPreferredMinSize(new Size(500, 1000));
         }
+
+        public SettingsPageViewModel ViewModel { get { return (SettingsPageViewModel)DataContext; } }
```

Comment:
Using Binding at the xaml file this row could be removed

Reply:
It's necessary for get properties in ViewModel.